src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs

```diff
@@ -866,7 +866,7 @@ public int LastIndexOf(T item, int index, int count)
             return Array.LastIndexOf(_items, item, index, count);
         }
 
-        // Removes the element at the given index. The size of the list is
+        // Removes the given element. The size of the list is
```

Comment:
```suggestion
        // Removes the first occurrence given element, if found. The size of the list is
```

Reply:
Yes, You are right @eiriktsarpalis

Reply:
@eiriktsarpalis Thank you for reviewing, I have made the changes, Please review now.